src/pages/Orders/index.tsx

```diff
@@ -1,12 +1,41 @@
 import React from 'react';
-import { useTranslation } from 'react-i18next';
+import OrderItem from '@/pages/Orders/components/OrderItem/OrderItem';
+import SortingRow from '@/pages/Orders/components/SortingRow/SortingRow';
+import { Box, Pagination } from '@mui/material';
+import { useOrdersPagination } from './useOrdersPagination';
 
 function OrdersPage(): React.ReactElement {
-  const { t } = useTranslation();
+  const { viewData, currentPage, totalPages, fetchOrders } =
```

Comment:
rename viewData to something more self-explainable,like: oders, odersList, ordersDetails

Reply:
renamed